how_to_use_package.md

```diff
@@ -0,0 +1,47 @@
+# 🚀 Quick Start Guide for matplotlib_visual
+--- 
+
+## **🔧Installations:**
+```bash
+pip install package
+```
+> **Note:** The `matplotlib-visual` package has been implemented by [Project Manager](https://github.com/this-is-yaash) and has details specific to him, you are sugested to follow the steps above and fork the repository and upload a package.
+
+## **⏱️Execution:**
+1. Create a Python file with **.py** extension and enter any of the code snippets below.
+2. Or, open a terminal and run Python interactively:
+```bash
+python
+```
+
+### 📊Features of this package:
+- **📍Displaying source code** in the console output.
```

Comment:
Displays source code, not displaying. 
in the console , not console output. We know the command will give an output.